### ISSUE TITLE
Listview: Second listview has wrong color

### ISSUE THREAD

**Issue by TomaszLukawskiSam** [Samsung/TAU#272]
UIComponents/components/list/expandable/expandable.html#

![image](https://user-images.githubusercontent.com/29534410/62283226-5d2b9a00-b451-11e9-9f5d-70577485cbad.png)

**Comment by lslachciak** [comment]
I think we have similar issue on 1.0:
![image](https://user-images.githubusercontent.com/2246331/62286736-a3d0c280-b458-11e9-97db-7a6a2f2a45aa.png)

https://samsung.github.io/TAU/examples/1.0/examples/mobile/UIComponents/components/list/expandable/expandable.html

**Comment by TomaszLukawskiSam** [comment]
The issue seems not trivial. 
There is three lists inside one list.

``` Listview.js (line: 624)
// check other sibling colored lists
siblingLists = [].slice.call(self.element.parentElement.querySelectorAll(WIDGET_SELECTOR));
```
Purpose of this code was find sibling lists but in this case, found lists are not sibling of main list.

**Comment by TomaszLukawskiSam** [comment]
The patch has already merged to TAU 1.0
